src/System.Management.Automation/minishell/api/RunspaceConfigurationTypeException.cs

```diff
@@ -20,12 +20,7 @@ namespace System.Management.Automation.Runspaces
     ///     2. ISerializable
     /// -->
     [Serializable]
-#if CORECLR
-    internal
-#else
-    public
-#endif
-    class RunspaceConfigurationTypeException : SystemException, IContainsErrorRecord
+    internal class RunspaceConfigurationTypeException : SystemException, IContainsErrorRecord
```

Comment:
Sorry, there are too many files (>150) and the only way to not confuse is to handle them one at a time.